modules/javafx.graphics/src/main/java/javafx/scene/layout/Border.java

```diff
@@ -389,6 +389,20 @@ public Border(@NamedArg("strokes") BorderStroke[] strokes, @NamedArg("images") B
         hash = result;
     }
 
+    /**
+     * A convenience factory method for creating a solid border with a single {@code Paint}. The border will use the
+     * default {@code CornerRadii} and the default {@code BorderWidths} of {@link BorderStroke}.
+     *
+     * @implSpec
+     * This call is equivalent to {@code new Border(new BorderStroke(stroke, BorderStrokeStyle.SOLID, null, null));}.
+     * @param stroke the stroke of the border
```

Comment:
`@param stroke The stroke to use for all sides. If {@code null}, defaults to {@code Color.BLACK}.`

Picked from BorderStroke doc